src/hca_validation/lambda_functions/entry_sheet_validator_lambda/handler.py

```diff
@@ -121,10 +150,11 @@ def handler(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
                     body = event['body']
                     
                 sheet_id = body.get('sheet_id')
+                bio_network = body.get('bio_network')
```

Comment:
```suggestion
                bio_network = body.get('bionetwork')
```

Could also update the variable names for consistency but of course the request data is the main important thing